.storybook/preview.js

```diff
@@ -6,7 +6,7 @@ import {withProviderSwitcher} from './custom-addons/provider';
 // decorator order matters, the last one will be the outer most
 
 configureActions({
-  depth: 1
+  depth: 10
```

Comment:
why? this was set to 1 to reduce the amount serialized and set from iframe to the actions panel where it needs to be parsed and displayed, which made some stories appear to be quite laggy

Reply:
Seems that this no longer works with the upgraded storybook version. It was erroring trying to parse something like `'[Array(Object)]'` as JSON. Seems the depth needs to be enough so that the arguments of the action are serialized at all.